htdocs/admin/workflow.php

```diff
@@ -137,7 +148,15 @@
 		'position' => 66,
 		'enabled' => ! empty($conf->expedition->enabled) && ! empty($conf->facture->enabled),
 		'picto' => 'shipment'
-	)
+	),
+
+	// Automatic classification of projects
+	'WORKFLOW_PROJECT_CLASSIFY_CLOSED_WHEN_ALL_TASKS_DONE' => array(
```

Comment:
The "workflow" module is dedicated to manage interaction between different modules.
But here project and class are in the same module. So please move the option into the setup of module project.